java/ql/src/semmle/code/java/Expr.qll

```diff
@@ -60,6 +60,12 @@ class Expr extends ExprParent, @expr {
   /** Gets the statement containing this expression, if any. */
   Stmt getEnclosingStmt() { statementEnclosingExpr(this, result) }
 
+  /**
+   * Gets a statement that transitively contains this expression, if any.
```

Comment:
Might have to be "directly or transitively". Transitively on its own would be:
```
this.getEnclosingStmt().getEnclosingStmt+()
```

Reply:
```suggestion
   * Gets a statement that directly or transitively contains this expression, if any.
```
That is indeed a bit clearer, thanks.